commcare/src/test/java/org/motechproject/commcare/web/FullFormControllerTest.java

```diff
@@ -94,7 +94,7 @@ public void testIncomingDeviceReport() throws EndpointNotSupported {
         assertEquals("2012-07-21T15:22:34.046462Z", parameters.get(RECEIVED_ON));
 
         Map<String, String> attributes = (Map<String, String>) parameters.get(ATTRIBUTES);
-        assertEquals(1, attributes.size());
+        assertEquals(2, attributes.size());
```

Comment:
please adjust this test, add app_id value and check if attributes map has this value